Source/WebCore/loader/EmptyClients.cpp

```diff
@@ -471,6 +471,7 @@ class EmptySearchPopupMenu : public SearchPopupMenu {
 
 private:
     PopupMenu* popupMenu() final { return m_popup.ptr(); }
+    RefPtr<PopupMenu> protectedPopupMenu() final { return m_popup.ptr(); }
```

Comment:
FYI, using `m_popup.copyRef()` would be slightly more efficient. It avoids the RefPtr constructor having the null check the pointer, since we know it is non null.

But also, we don't need this override function at all.